select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -53,7 +55,6 @@
 	private String initSelection;  //TODO this will not work
 	private String query;
 	private String width;
-	private String theme;
```

Comment:
I think it wont: there is still a setter receiving an String (the name of the theme). To the end user noting changes.